Products/Poi/content/tracker.py

```diff
@@ -389,7 +389,7 @@ def getTagsInUse(self):
         issues = self.getIssues()
         tags = {}
         for i in issues:
-            for s in i.subject:
+            for s in i.Subject:
```

Comment:
I’m sure this works or you wouldn’t have changed it, but why? The name is lower case in the interface class.

See https://github.com/collective/Products.Poi/blob/3.0-development/Products/Poi/content/issue.py#L140